minetest.conf.example

```diff
@@ -603,4 +603,5 @@
 #    Comma-separated list of trusted mods that are allowed to access insecure
 #    functions even when mod security is on (via request_insecure_environment()).
 #secure.trusted_mods =
-
+#	enable mod manager 
```

Comment:
This is the wrong place, it's a client setting. Put it next to the other main menu settings (such as mainmenu_last_idx or something)